eq-author/src/App/questionPage/Design/index.test.js

```diff
@@ -438,10 +439,32 @@ describe("QuestionPageRoute", () => {
         .simulate("click");
 
       expect(mockHandlers.onDuplicatePage).toHaveBeenCalledWith({
-        sectionId: match.params.sectionId,
+        sectionId: page.section.id,
```

Comment:
How is this test not failing? Are they both `undefined`?

Reply:
Good question. I added in an extra test to see what was happening:

Expected mock function to have been called with:
      {"ignoreMe": 1, "pageId": undefined, "position": 1, "sectionId": undefined}

 as argument 1, but it was called with
      {"pageId": undefined, "position": 1}.

Looks like Jest is ignoring undefined parameters.

I've removed sectionId from test for clarity